### PR TITLE
docs(CH): update existing seq diagram references

### PR DESCRIPTION
**Pull request by 0xangelo**
Includes:
* links to the diagram files in the new dedicated repo (before it had references to the old monorepo)
* add missing diagrams for `close_market` and `settle_position` extrinsics
* removes an unneeded storage write

### CODE REVIEW

**Review by nickkuk — APPROVED** [review]

File: clearing-house/src/lib.rs [hunk 1]
```diff
@@ -620,7 +620,7 @@ pub mod pallet {
         /// alludes to the amount of collateral that can be withdrawn without making the account go
         /// below the initial margin ratio.
         ///
-        /// ![](https://www.plantuml.com/plantuml/svg/FOux3i8m40LxJW47IBQdYeJ4FJREmxPahwtzmFM9AAX6CzLivgmUlLrkLLAB0w7jMjodtOcKFskkNc8FWwOX3l4rZKwFqUSmtXkUbT9V29OAb5xA7PGQMAlafOmmq54vdzr8qSSRIsVDLTRPM7u76-Gu-GK0)
+        /// ![](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/escher-exchange/escher/main/clearing-house/diagrams/withdraw-collateral.plantuml)
```
Comment (Cardosaum) [inline clearing-house/src/lib.rs:623]: I was wondering, is it worth to upgrade all links to `https` ?

**Review by Cardosaum — APPROVED** [review]
LGTM

File: clearing-house/src/lib.rs (hunk 1 above)
Comment (0xangelo) [reply clearing-house/src/lib.rs:623]: I don't think there's any sensitive information we need to protect in this case (we're just downloading an image)

Comment (Cardosaum) [reply clearing-house/src/lib.rs:623]: Fair enough :+1:

**Pull request merged**